common/src/allocations/monitor.rs

```diff
@@ -148,17 +147,18 @@ mod test {
     async fn test_parses_allocation_data_from_network_subgraph_correctly() {
         let (network_subgraph, _mock_server) = setup_mock_network_subgraph().await;
 
-        let allocations = indexer_allocations(
+        let result = get_allocations(
             network_subgraph,
             *test_vectors::INDEXER_ADDRESS,
-            Duration::from_secs(60),
             Duration::from_secs(300),
-        );
-
-        assert_eq!(
-            allocations.value().await.unwrap(),
-            *test_vectors::INDEXER_ALLOCATIONS
-        );
+        )
+        .await;
+        match result {
+            Ok(allocations) => {
+                assert_eq!(allocations, *test_vectors::INDEXER_ALLOCATIONS)
+            }
+            Err(e) => println!("Error: {:#?}", e),
+        }
```

Comment:
This test won't fail in case of an error. Try to use `assert`s and `panic` instead of println